src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -226,6 +226,7 @@
 #editor-zone {
   width: auto !important;
   padding: 0 !important;
+  height: 100% !important;
```

Comment:
Resolves an awkward calculation made by canvas tools and allow VoTT to regain control over height of element.